test/e2e/apimachinery/webhook.go

```diff
@@ -2166,19 +2132,25 @@ func registerSlowWebhook(f *framework.Framework, configName string, certCtx *cer
 	})
 	framework.ExpectNoError(err, "registering slow webhook config %s with namespace %s", configName, namespace)
 
-	err = waitWebhookConfigurationReady(f)
+	err = waitWebhookConfigurationReady(ctx, f)
 	framework.ExpectNoError(err, "waiting for webhook configuration to be ready")
 
-	return func() {
-		client.AdmissionregistrationV1().ValidatingWebhookConfigurations().Delete(context.TODO(), configName, metav1.DeleteOptions{})
+	cleanup := func(ctx context.Context) {
+		err := client.AdmissionregistrationV1().ValidatingWebhookConfigurations().Delete(ctx, configName, metav1.DeleteOptions{})
+		if !apierrors.IsNotFound(err) {
+			framework.ExpectNoError(err)
+		}
 	}
```

Comment:
it is more verbose but it is easier to understand that the internals of `framework.IgnoreNotFound(` using reflection and type assertion